astropy/coordinates/earth.py

```diff
@@ -104,6 +104,8 @@ class EarthLocationInfo(QuantityInfoBase):
     """
     _represent_as_dict_attrs = ('x', 'y', 'z', 'ellipsoid')
 
+    mask_val = np.ma.masked
```

Comment:
I'm a bit surprised that with #11787 this is still necessary. Does this perhaps pre-date it?

Reply:
Hmmm.  I started this from the #11787 branch and made that change in response to some failure. However, after removing it now tests still pass, so there you go. I'm taking it out.

Reply:
Reverted this.